plotly/tools.py

```diff
@@ -1501,6 +1508,35 @@ def validate_ohlc(open, high, low, close, direction, **kwargs):
                                          "'both'")
 
     @staticmethod
+    def validate_distplot(hist_data, curve_type):
+        """
+        distplot specific validations
+
+        :raises: (PlotlyError) If hist_data is not a list of lists
+        :raises: (PlotlyError) If curve_type is not valid (i.e. not 'kde' or
+            'normal').
+        """
+        hist_data_types = (list,)
+        if _numpy_imported:
+            hist_data_types += (np.ndarray,)
+
+        if not isinstance(hist_data[0], hist_data_types):
+                raise exceptions.PlotlyError("Oops, this function was written "
+                                             "to handle multiple datasets, if "
+                                             "you want to plot just one, make "
+                                             "sure your hist_data variable is "
+                                             "still a list of lists, i.e. x = "
+                                             "[1, 2, 3] -> x = [[1, 2, 3]]")
```

Comment:
ahhhh. this'll fail with pandas series. The easiest thing to do is probably add:

```
if _pandas_imported:
    hist_data_types ++ (pd.Series, )
```